### ISSUE TITLE
Usage is confusing for a newcomer

### ISSUE THREAD

**Issue by kaba2** [danielbrodin/atom-project-manager#319]
Hi there! I am a newcomer to Atom from Sublime Text. I understand Atom does not have native support for projects yet, and that this package tries to cover on that. Good work on helping us with this package on the meantime! Here are some suggestions on how to improve the frontpage documentation.

### Problem

The documentation on the frontpage is confusing to me, and as a result I haven't been successful in getting the package to work this far. What I want is to have a local project file on my project directory, to be stored with version control, as I can do in Sublime Text. 

### Frontpage documentation

The frontpage reads:

    All these settings can be added to a project.cson file in the root folder of the project. It follows the below example, but without the array.

    [
      {
        title: 'Project Manager'
        group: 'Atom'
        paths: [
          '/path/to/project-manager'
        ]
        devMode: true
        settings:
          'editor.tabLength': 4
          'editor.showInvisibles': true
      }
    ]

I gather that I need to create project file on the root directory of my project. The project file should resemble the above example. 

### Questions

The following questions are meant to describe my thinking process.

* Must `project.cson` be named exactly like that, or can I name it like `foobar.cson`? 
* If the latter, how does the project manager know which `.cson` file to use? 
* In fact, the project manager does not seem to find any project after creating a `project.cson` file.  Am I missing something? Do the local project files work at all?
* What does "the array" refer to?
* If "the array" refers to the outermost array in the cson file, and it should not be there, then why is it part of the example?
* Using the package-manager itself as an example project confuses me on whether some of these settings need to be there to get the project to work. In particular, `'/path/to/project-manager'` suggests that I should replace it with the physical path to the package manager on my hard disk.
* I guess that `paths` should list the set of of paths for the project to include.
* Can I use a path relative to the project file (e.g. `.`), or do the paths need to be absolute paths, as in the example? Requiring an absolute path would be bad, because then I cannot move the project on my hard disk.
* Why do I need to specify a path at all? Could the project root directory not be taken by default to be the same as the directory where the project file resides?

In summary, I would like to see a clear "Hello, world!" example for the package manager, using a local project file. It would be great if the frontpage documentation was improved so that the above questions were answered, and the ambiguities removed!

**Comment by danielbrodin** [comment]
Thanks for your input. Will see if I get some time to update the readme in the future. You are always welcome to create a PR if you have anything you know you can make more clear :)
